src/app/access-requests/features/access-request-manager-detail/access-request-manager-detail.component.html

```diff
@@ -233,65 +248,79 @@ <h2 mat-dialog-title>Access Request Detail</h2>
         </mat-card>
       </div>
     </div>
-  </mat-dialog-content>
-  <mat-dialog-actions class="flex gap-x-2 gap-y-4 border-gray-500 pt-4">
-    @if (ivasAreLoading() || ivasError() || !ivas().length || !changeable()) {
-      <button
-        mat-stroked-button
-        (click)="cancel()"
-        data-umami-event="Access Request Manager Close Clicked"
-      >
-        Close
-      </button>
-    } @else {
-      <div class="w-full sm:w-auto">
+
+    <div class="mt-8 flex">
+      @if (ivasAreLoading() || ivasError() || !ivas().length || !changeable()) {
         <button
-          autoFocus
           mat-stroked-button
-          class="w-full sm:w-auto"
-          (click)="cancel()"
-          data-umami-event="Access Request Manager Cancel Clicked"
+          (click)="goBack()"
+          data-umami-event="Access Request Manager Close Clicked"
         >
-          Cancel
+          Close
```

Comment:
```suggestion
          Cancel
```

Reply:
I changed this to "Close" because the page makes the changes immediately (like when you select an IVA or click on of the green checkmarks), while a "Cancel" would somewhat imply that these changes are rolled back.

Maybe we should label it "Cancel" only if one of the field editors is open and has a pending change in its input field, and label it "Close" otherwise? Another option would be to add a navigation guard that would show a confirmation dialog before closing the page and navigating away in this state. I think one of these two measures would be sufficient. Which one do you like better?

Reply:
It really is not a 'closing' action though. Maybe 'return' or 'back' would be better?

Reply:
Particularly if we make it an actual back button, then "Back" would be a better than "Close". Should we do that?

Regarding pending changes, I still have the question whether we should add a guard or just rename the button to "Cancel" while they are pending.